osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -81,6 +83,41 @@ private void load()
                 foreach (var drawableHitObject in NestedHitObjects)
                     drawableHitObject.AccentColour.Value = colour.NewValue;
             }, true);
+
+            Tracking.BindValueChanged(updateSlidingSample);
+        }
+
+        private SkinnableSound slidingSample;
+
+        protected override void LoadSamples()
+        {
+            base.LoadSamples();
+
+            slidingSample?.Expire();
```

Comment:
Is this a performance optimisation to stop updates?

Reply:
Check where this is called (generally on sample change, editor for now). A new instance is being constructed just below. We need to kill the last one.

Reply:
Aaaaaah I misread the method as `LoadComplete`